theworkoutinfo/Model/Exercise.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Exercise.swift
+//  theworkoutinfo
+//
+//  Created by Gonzalo Barrios on 4/4/18.
+//  Copyright © 2018 Gonzalo Barrios. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class Exercise : Mappable {
+    
+    let id: String
+    let name: String?
+    let description: String?
+    var imageURL: String? = nil
+    
+    required init?(json: JSON) {
+        guard let id = json["id"].int else {
+            return nil
+        }
+        
+        self.id = "\(id)"
```

Comment:
If the id in the API is Int, can we use Int in the model too instead of casting to String?